### DISCUSSION TITLE
MP3 - Needs higher bit rate

### DISCUSSION THREAD

**Discussion by SzuDar** [aclap-dev/video-downloadhelper#1548] [Help and Q&A]
### Your question

I need to download mp3 with a bit rate higher than 128kb/s. Where in the program settings can I change this? 

### Addon & CoApp version and details

version: 9.1.0.44
target: mozilla
channel: stable
lang: pl
coapp: {"found":true,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19","new_version":false}
license: {"accepted":true,"email":"<removed>","key":"13760f022e8e4540"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:133.0) Gecko/20100101 Firefox/133.0
{
    "valid_license_message_has_been_displayed": true,
    "used_history_button": true,
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "use_sidebar": false,
    "open_count_store": 2
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

There is no setting in VDH to control this.  The bit rate is controlled by the author of the content.  It might also be controlled by the hosting web site, but it's more likely the author posts the content with whatever bit rate the author chose.  VDH downloads what's there.  If it's only 128k, that's what you get.  If it's 320k, that's what you get.  This is not a result of anything VDH is doing.  The source material is what it is.  If it's no higher than 128k, then that's what you get.

If you post some URLs of examples, I might be able to show you what I'm talking about.  Maybe.  Not all content is susceptible to the techniques I would have to use to discover the properties of the input.  But it's might be worth a shot.
